source/specify-fields-return.txt

```diff
@@ -0,0 +1,155 @@
+.. _java-rs-project:
+
+========================
+Specify Fields To Return
+========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, filter, project, select, choose
+
+Overview
+--------
+
+In this guide, you can learn how to specify which fields to return from a read
+operation by using a **projection**. A projection is a document that specifies
+which fields MongoDB returns from a query.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<java-rs-getting-started>` guide.
+
+.. include:: /includes/reactor-note.rst
+
+Projection Types
+----------------
+
+You can use a projection to specify which fields to include in a return
+document, or to specify which fields to exclude. You cannot combine inclusion and
+exclusion statements in a single projection, unless you are excluding the
+``_id`` field.
+
+Specify Fields to Include
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the following syntax to specify the fields to include in the result:
+
+.. code-block:: java
+
+   projection(fields(include("<field name>")))
+
+To use a projection, pass your query filter to the ``find()`` method. Then, chain the ``projection()`` method to the ``find()`` method. 
+The following example uses the ``find()`` method to find all restaurants with
+the ``name`` field value of ``"Emerald Pub"``. It then uses a projection to
+include only the ``name``, ``cuisine``, and ``borough`` fields from the returned documents.
+
+.. io-code-block::
+    
+   .. input:: /includes/project.java
+      :start-after: start-project-include
+      :end-before: end-project-include
+      :language: java
+
+   .. output::
+
+      {'_id': ObjectId('...'), 'borough': 'Manhattan', 'cuisine': 'American', 'name': 'Emerald Pub'}
+      {'_id': ObjectId('...'), 'borough': 'Queens', 'cuisine': 'American', 'name': 'Emerald Pub'}
+
+When you use a projection to specify fields to include in the return
+document, the ``_id`` field is also included by default. All other fields are
+implicitly excluded. To remove the ``_id`` field from the return
+document, you must :ref:`explicitly exclude it <java-rs-project-remove-id>`.
+
+.. _java-rs-project-remove-id:
+
+Exclude the ``_id`` Field
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When specifying fields to include, you can also exclude the ``_id`` field from
+the returned document.
+
+The following example performs the same query as the preceding example, but
+excludes the ``_id`` field from the projection:
```

Comment:
```suggestion
The following example performs the same query as in the preceding example, but
also specifies exclusion of the ``_id`` field in the returned documents:
```

Reply:
I kept it as "also excludes"; specifies exclusion feels very formal to me/a little awkward.